universal-application-tool-0.0.1/app/services/applicant/ReadOnlyApplicantProgramServiceImpl.java

```diff
@@ -68,7 +68,8 @@ public String getProgramTitle() {
           getBlocks(
               block ->
                   (!block.isCompleteWithoutErrors()
-                          || block.wasCompletedInProgram(programDefinition.id()))
+                          || block.wasCompletedInProgram(programDefinition.id())
+                          || block.containsStatic())
```

Comment:
nit - consider adding a comment explaining the rationale for this behavior of static blocks since it might not be entirely obvious to future code maintainers.